app/styles/app/layouts/insights.scss

```diff
@@ -0,0 +1,284 @@
+$axis-line-color: $dry-cement;
+$axis-tick-color: $cement-grey;
+$axis-label-color: $asphalt-grey;
+
+$grid-line-color: $pebble-grey;
+$grid-background-color: #fdfdfd;
+
+$med-small-up: "only screen and (min-width: 38em)";
+
+.insights-controls {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0 15px;
+
+  &-left {
+    display: grid;

```

Comment:
Hmm for some reason I had forgotten how limited IE's support for grid was, and how limited autoprefixer is in transforming current grid syntax to the old one.

It ended up looking _okay_ in IE, but only because autoprefixer doesn't try to convert to old grid syntax by default, and so everything just ended up as blocks haha. Anyway, in this case it's pretty easy to convert to use flexbox so I just did that.

Reply:
That's great, thank you!